templates/style/_navbar.scss

```diff
@@ -349,6 +349,14 @@ div.nav-container {
         li a {
             overflow-x: hidden;
             text-overflow: ellipsis;
+
+            &.current {
+                font-weight: bold;
+                display: list-item;
+                list-style-type: disc;
+                list-style-position: inside;
```

Comment:
Instead of creating a character or using an image, I preferred to rely on browser's provided CSS elements.